book/src/proposals/booting-from-snapshot.md

```diff
@@ -0,0 +1,262 @@
+# Booting From a Snapshot
+
+## Problem
+
+A validator booting from a snapshot should confirm that they are caught up to
```

Comment:
I don't like seeing "should" or "need" in problem statements. A problem is one of the things that occurs when they don't do what they supposedly should do.